src/CMakeLists.txt

```diff
@@ -46,12 +48,20 @@ target_compile_definitions(projectMSDL
         PROJECTMSDL_VERSION="${PROJECT_VERSION}"
         )
 
+if(APPLE)
+    include(fetch_syphon.cmake)
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -ObjC++")
+    find_library(SYPHON Syphon ${PROJECT_SOURCE_DIR})
+    target_link_libraries(projectMSDL PRIVATE ${SYPHON})
+endif()
+
 target_link_libraries(projectMSDL
         PRIVATE
         libprojectM::playlist
         Poco::Util
         SDL2::SDL2$<$<STREQUAL:${SDL2_LINKAGE},static>:-static>
         SDL2::SDL2main
+        glad
```

Comment:
Is glad required for all platforms, even macOS, when not building Syphon support? I'd suggest making All the Syphon-specific code, libraries etc. dependent on a specific `ENABLE_SYPHON` option or similar, then check for all required dependencies and requirements and give the user proper feedback of something's wrong or missing.

Also, use `cmake_dependent_option()` to only show the option to devs if the build is executed on macOS, as it doesn't make sense or works on any other platform.